packages/@lightningjs/ui-components/src/components/Tile/Tile.d.ts

```diff
@@ -18,10 +18,20 @@
 
 import lng from '@lightningjs/core';
 import { MetadataBaseStyle } from '../MetadataBase';
```

Comment:
Just drawing your attention to the warning that this should be removed.

Reply:
but are we sure it's never used? /s

Reply:
@arwehrman why do we need MetadataBaseStyle to be imported? if it's not being used by anything we should remove the import

Reply:
removed